tests/fs/tree.test.ts

```diff
@@ -24,10 +24,10 @@ describe("FS", () => {
 
       return [
         await fs.read(privatePath),
-        await fs.read(publicPath)
+        new TextDecoder().decode(await fs.read(publicPath))
```

Comment:
This didn't fail before, because `Buffer`s would return their content interpreted as a utf8 string if forced to be turned to a string (e.g. via `[<buffer>].join("")`).
But we really shouldn't depend on that behavior.

At some point we should really add an encoding parameter to webnative's filesystem operations similar to how it works in many other APIs (i.e. `fs.read(path, { encoding: 'utf8' })`, and `fs.write(path, content, { encoding: 'utf8' })`).